test/jit/test_builtins.py

```diff
@@ -109,22 +109,31 @@ def fn(x):
                 return a
 
     def test_del_multiple_operands(self):
+        def fn(x):
+            # type: (List[int]) -> List[int]
+            a, b, c = x[0], x[1], x[2]
+            del a, b, c
+            return x
 
-        with self.assertRaisesRegex(torch.jit.frontend.NotSupportedError,
-                                    "with more than one operand"):
-            @torch.jit.script
-            def del_list_multiple_operands(x):
-                # type: (List[int]) -> List[int]
-                del x[0], x[1]
-                return x
+        self.checkScript(fn, ([1, 2, 3],))
 
-        with self.assertRaisesRegex(torch.jit.frontend.NotSupportedError,
-                                    "with more than one operand"):
-            @torch.jit.script
-            def del_dict_multiple_operands(x):
-                # type: (Dict[str, int]) -> Dict[str, int]
-                del x['hi'], x['there']
-                return x
+        def del_list_multiple_operands(x):
+            # type: (List[int]) -> List[int]
+            del x[0], x[1]
+            return x
+
+        py_out = del_list_multiple_operands([0, 1, 2])
+        jit_out = torch.jit.script(del_list_multiple_operands)([0, 1, 2])
+        self.assertEquals(py_out, jit_out)
```

Comment:
Use `self.checkScript`?

Reply:
You can't - list inputs are mutable. A list input will be mutated by the script function and then reused by the eager mode function.